packages/valory/skills/transaction_settlement_abci/behaviours.py

```diff
@@ -627,7 +627,7 @@ class CheckLateTxHashesBehaviour(  # pylint: disable=too-many-ancestors
     check_expected_to_be_verified = "One of the next tx checks"
 
     @property
-    def history(self) -> List[str]:
+    def history(self) -> List[str]:  # pragma: no cover
```

Comment:
used nowhere

Reply:
discuss with @Adamantios if this is safe to remove

Reply:
We need to cover this.